Failures running Javac for DDR_Test

PR builds are failing regularly trying to run the DDR_Tests. Failures report a compile step failed related to javac with cryptic messages similar to:
```10:48:33 /home/u0020236/workspace/PullRequest-Sanity-JDK11-aix_ppc-64_cmprssptrs-OpenJ9/openj9/test/functional/DDR_Test/build.xml:59: Error running /home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK11-aix_ppc-64_cmprssptrs-OpenJ9/build/aix-ppc64-normal-server-release/images/jdk/bin/javac compiler
10:48:33 
10:48:33 Total time: 1 second
10:48:33 makefile:82: recipe for target 'compile' failed
10:48:33 make: *** [compile] Error 1
[Pipeline] }
```

see https://ci.eclipse.org/openj9/job/PullRequest-Sanity-JDK11-aix_ppc-64_cmprssptrs-OpenJ9/85/console

--- a/buildenv/jenkins/jobs/builds/Build-Test-Any-Platform
+++ b/buildenv/jenkins/jobs/builds/Build-Test-Any-Platform
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2018 IBM Corp. and others
+ * Copyright (c) 2018, 2019 IBM Corp. and others
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
@@ -43,9 +43,6 @@ timeout(time: 10, unit: 'HOURS') {
                 }
 
                 buildFile = load 'buildenv/jenkins/common/build'
-                if (run_tests()) {
-                    testFile = load 'buildenv/jenkins/common/test'
-                }
             } finally {
                 // Temporarily turn off disableDeferredWipeout. See https://issues.jenkins-ci.org/browse/JENKINS-54225
                 cleanWs notFailBuild: true, disableDeferredWipeout: false
@@ -59,6 +56,9 @@ timeout(time: 10, unit: 'HOURS') {
                 // clean up
                 buildFile.build_all()
             } else {
+                checkout scm
+                testFile = load 'buildenv/jenkins/common/test'
+                cleanWs()
                 buildFile.build_pr()
                 testFile.test_all()
             }

